webpack.config.js

```diff
@@ -50,18 +53,29 @@ const generateConfig = ({extensionPath, devMode=false, customOutputPath, analyze
   const pluginCompress = new CompressionPlugin({
     filename: "[path].gz[query]",
     algorithm: "gzip",
-    test: /\.js$|\.css$|\.html$/,
-    threshold: 10240,
-    minRatio: 0.8
+    test: /\.(js|css|html)$/,
+    threshold: 4096
+  });
+  const pluginHtml = new HtmlWebpackPlugin({
+    filename: 'index.html',
+    template: './index.html'
```

Comment:
Since the template is unusable by itself, would it be more typical to move this to `src/index.html`?

Reply:
Yep I think that could be a good idea 👍

Reply:
The current change means that this doesn't build on nextstrain.org

```
This doesn't build when auspice is run as a project's dependency with extensions. For instance, on nextstrain.org we get
```
internal/fs/utils.js:230
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/Users/naboo/github/nextstrain/nextstrain.org/node_modules/auspice/index.html'
    at Object.openSync (fs.js:457:3)
    at Object.readFileSync (fs.js:359:35)
    at Object.exportIndexDotHtml (/Users/naboo/github/nextstrain/nextstrain.org/node_modules/auspice/cli/utils.js:112:17)
    at /Users/naboo/github/nextstrain/nextstrain.org/node_modules/auspice/cli/build.js:57:15
 ...
```

I think this is relatively straightforward to fix by updating `exportIndexDotHtml()`

Reply:
Actually with index.html being outputted in the /dist folder, exportIndexDotHtml() becomes completely obsolete. As far as I can see, so I think it can be killed.

Reply:
In the interest of keeping the risk of breaking other moving parts at a minimum, it's still probably better to updated exportIndexDotHtml. This new updated version I pushed is compatible with nextstrain.org `master` as well.